src/sentry/api/serializers/models/apitoken.py

```diff
@@ -4,7 +4,7 @@
 
 @register(ApiToken)
 class ApiTokenSerializer(Serializer):
-    def get_attrs(self, item_list, user):
+    def get_attrs(self, item_list, user, **kwargs):
```

Comment:
Do we need to capture the kwargs here?

Would rather be explicit in the api args, rather than allowing any kwargs to be passed

Reply:
Unfortunately no, we cannot make a change the would break the contract of the base class function https://mypy.readthedocs.io/en/stable/common_issues.html#incompatible-overrides

`Other incompatible signature changes in method overrides, such as adding an extra required parameter, or removing an optional parameter, will also generate errors.`

This fails on our linter as well, as I tried that initially https://github.com/getsentry/sentry/actions/runs/7227889927/job/19696388755

![Screenshot 2023-12-18 at 3 01 44 PM](https://github.com/getsentry/sentry/assets/5581484/1c8ff58b-5fad-4a08-83d4-d50aeae6550e)